prompts/conversation/summarize/prompt2.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
this is actually a prompt for memory, i would maybe put in its own memory folder